stdlib/public/runtime/StackAllocator.h

```diff
@@ -227,9 +237,10 @@ class StackAllocator {
   };
 
   // Return a slab which is suitable to allocate \p size memory.
+  SWIFT_ALWAYS_INLINE
   Slab *getSlabForAllocation(size_t size) {
     Slab *slab = (lastAllocation ? lastAllocation->slab : firstSlab);
-    if (slab) {
+    if (SWIFT_UNLIKELY(slab)) {
```

Comment:
This seems likely.

Reply:
I blame daylight saving, end of. It actually makes a decent difference in the codegen. Fixing this makes the fast path quite a bit more linear. I was getting a little confused following it around before because it bounced back and forth so much, but didn't quite realize what that meant. Pushed the fix now.